asab/storage/mongodb.py

```diff
@@ -26,7 +26,12 @@ class StorageService(StorageServiceABC):
 	def __init__(self, app, service_name, config_section_name='asab:storage'):
 		super().__init__(app, service_name)
 		self.Client = motor.motor_asyncio.AsyncIOMotorClient(asab.Config.get(config_section_name, 'mongodb_uri'))
-		self.Database = self.Client[asab.Config.get(config_section_name, 'mongodb_database')]
+
+		self.Database = self.Client.get_database(
+			asab.Config.get(config_section_name, 'mongodb_database'),
+			codec_options=bson.codec_options.CodecOptions(tz_aware=True),
```

Comment:
`tz_aware` - it really means that the datetime is in UTC? Not local time ...

Reply:
```
tz_aware=True,
...     tzinfo=timezone.utc))
```

?

Reply:
Okay, I just read more on this so here is a clarification:

- There is **no way** to preserve the `tzinfo` attribute in datetime objects in Mongo.
- `tz_aware=True` only causes the **python client** to add `tzinfo` to all datetime objects retrieved from the database. Inside the database they are still timezone-naive.
- If no `tzinfo` parameter is specified, the retrieved objects get `tzinfo=timezone.utc` as default.